src/axom/mint/docs/sphinx/sections/citations.rst

```diff
@@ -6,13 +6,13 @@
 
 .. _LLNL: https://www.llnl.gov/
 .. _Umpire: https://umpire.readthedocs.io/en/develop/
-.. _Mint Doxygen API Documentation: https://lc.llnl.gov/axom/docs/doxygen/axom_doxygen/html/classaxom_1_1mint_1_1Mesh.html
-.. _Axom Quick Start Guide: https://axom.readthedocs.io/en/develop/docs/sphinx/quickstart_guide/index.html
+.. _Mint Doxygen API Documentation: ../../../../../doxygen/html/minttop.html
```

Comment:
Building the documentation gives complaints "duplicate explicit targets" for lines 9, 10, 14, and 15.  Does this file actually get linked to?  @gzagaris , could we delete this file?
Edit: See also slic's file citations.rst.

Reply:
We shouldn't delete this file. See comment below.